docs/sources/reference/commandline/cli.md

```diff
@@ -1225,6 +1225,9 @@ For example:
     EventsListeners: 0
     Init Path: /usr/bin/docker
     Docker Root Dir: /var/lib/docker
+    Http Proxy: http://test:test@localhost:8080
+    Https Proxy: https://test:test@localhost:8080
+    No Proxy: /var/run/docker.sock,9.81.1.160
```

Comment:
I think `/var/run/docker.sock` could remove from the `No Proxy`, since( #10739) has fixed this, there is no need to add `/var/run/docker.sock` to `No Proxy`, this may misleading the user at some point